GoogleForCreators/web-stories-wp
# Media: Fix empty button after cropping

## Context



## Summary

Override the method to render the button toolbar, so that button name is correctly overridden. 

Basically what happened here, is the override to change the name was stored in state and not in the component, when toolbar re-render, the state was empty. 

## Relevant Technical Choices



## To-do



## User-facing changes

https://user-images.githubusercontent.com/237508/180421234-e1bb95b7-14f1-4a49-a128-a24badd268b8.mov

## Testing Instructions






This PR can be tested by following these steps:

1. Open editor. 
2. Upload media dialog.
3. Select image. 
4. Click edit image. 
5. Edit image. 
6. Save. 
7. Click button. 
8. See button has text. 

#### Poster. 
1. Open editor.
2. Select documents tab. 
3. add poster. 
4. See dialog has button has text. 


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #11968
